modules/runtime-info-ui/pom.xml

```diff
@@ -136,6 +136,25 @@
               </resources>
             </configuration>
           </execution>
+          <execution>
+            <id>copy-rest-doc-libs</id>
+            <phase>compile</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/target/classes/ui/js/</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>node_modules/</directory>
+                  <includes>
+                    <include>jquery/dist/jquery.min.js</include>
```

Comment:
Probably that means you need to adapt the index page of runtime-info-ui (not the rest_docs page)